### PR TITLE
Gfx voronoi

### PR DESCRIPTION
**Pull request by br-olf**
I wrote a new effect :)

### CODE REVIEW

File: src/modules/gfx_voronoi.c [hunk 1]
```diff
@@ -0,0 +1,357 @@
...
+    }
+    else if (norm_type == P6){
+        return p6_dist(x1,y1,x2,y2);
+    }
+    else if (norm_type == MULTIPLICATE){
+        return multiplicate_dist(x1,y1,x2,y2);
+    }
+    else{
+        // unreachable
+    }
```
Comment (vifino) [inline src/modules/gfx_voronoi.c:236]: No need to have an else when there is nothing in it.
```suggestion
```

File: src/modules/gfx_voronoi.c [hunk 2]
```diff
@@ -0,0 +1,357 @@
...
+static const norm_rand_min = 0;
+static const norm_rand_max = 7;
+static uint16_t choose_random_norm = 0;
+
+#define P_MAX 20
+static struct Point {
+	float x;
+	float y;
+    float vx;
+    float vy;
```
Comment (vifino) [inline src/modules/gfx_voronoi.c:66]: Mismatched indentation here. Maybe convert everything to tabs and check it's all sane? (Or spaces if you prefer that...)

**Review by vifino — CHANGES_REQUESTED** [review]
Hey! Lovely to see another effect from you! :)

However, I have a couple of changes I'd like you do consider:
I'll add them to lines if there is just one, but there are a couple extras.

1) Can you rebase on master? There are a lot of commits that have already been merged.

2) `p3`, `p4`, `p5` and `p6` dist functions all seem pretty similar and come down to `powf(dx, n) + powf(dy, n)` if I'm not mistaken. Could you maybe just use that instead?

3) In general, there are a few codestyle nit-picks, I'd like you to change the following:
  - Remove the blank lines following function definitions (like in `static void myinit()`)
  - Add a space between curly braces and function definitions or keywords (like line 239, 241, etc..)
  - Pick whether you want to have the curly braces on the same line or on the next, but please don't mix. Personally, I prefer them on the same line because it's not the 80's and we have wide monitors.
  - Unless there is a reason for doing so, could you have the `p` functions and the switch in order? :)

I know it's a lot, but it's all really minor stuff. If you disagree with some of the changes or don't feel like it, it's fine, but let me know.
Thank you!!

**Comment by br-olf** [comment]
I rebased and therefore that is the new PR: https://github.com/shinyblink/sled/pull/151

**Pull request closed without merging**